apps/openchallenges/organization-service/src/test/java/org/sagebionetworks/openchallenges/organization/service/model/mapper/OrganizationMapperTest.java

```diff
@@ -0,0 +1,111 @@
+package org.sagebionetworks.openchallenges.organization.service.model.mapper;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.sagebionetworks.openchallenges.organization.service.model.dto.OrganizationDto;
+import org.sagebionetworks.openchallenges.organization.service.model.entity.OrganizationEntity;
+
+public class OrganizationMapperTest {
+  @Test
+  public void ConvertToEntity_ShouldReturnDtoProperties_WhenDtoPropertiesPassed() {
+
+    // create new Dto with properties to be copied to organization entity
+    OrganizationDto user = new OrganizationDto();
+    user.setName("Test Organization");
+    user.setDescription("This is a test organization");
+    user.setId(4599L);
+    user.setEmail("testemail@test.edu");
+    user.setLogin("login");
+    user.setAvatarKey("avatar Key");
+    user.setWebsiteUrl("url");
+    user.setChallengeCount(7);
+    user.setAcronym(null);
+
+    OrganizationMapper mapper = new OrganizationMapper();
+
+    OrganizationEntity organizationEntity = mapper.convertToEntity(user);
```

Comment:
You could simplify by renaming `organizationEntity` to `entity` as there is only one entity defined here. Also to be consistent with the naming you use for the mapper. Consider other places where this comment could apply.